sysdeps/managarm/generic/file.cpp

```diff
@@ -927,8 +927,8 @@ int sys_msg_recv(int sockfd, struct msghdr *hdr, int flags, ssize_t *length) {
 	managarm::fs::RecvMsgReply<MemoryAllocator> resp(getSysdepsAllocator());
 	resp.ParseFromArray(recv_resp.data(), recv_resp.length());
 
-	if (resp.error() == managarm::fs::Errors::WOULD_BLOCK) {
-		return EAGAIN;
+	if (resp.error() != managarm::fs::Errors::SUCCESS) {
+			return resp.error() | toErrno;
 	} else {
 		__ensure(resp.error() == managarm::fs::Errors::SUCCESS);
```

Comment:
This is a separate change, and this ensure is redundant now.

Reply:
Noted, but the change is listed in the commit message it's in. So should be fine right?